examples/SpritesheetTest.js

```diff
@@ -123,3 +127,19 @@ p.tick = function () {
 	var fileName = __dirname + '/output/test_' + (this.index++) + '.png';
 	fs.writeFileSync(fileName, this.canvas.toBuffer(), '');
 }
+
+/**
+ * Helper for reporting back exec errors
+ */
+function execCallback(error, stdout, stderr) {
```

Comment:
This helped reveal a problem in `ffmpeg` output. The cli tool has probably changed a bit since this work was originally done.